py-polars/polars/dataframe/frame.py

```diff
@@ -640,6 +640,18 @@ def plot(self) -> DataFramePlot:
           is shorthand for
           `alt.Chart(df).mark_attr(tooltip=True).encode(**kwargs).interactive()`
 
+        For configuration, we suggest reading
+        `Chart Configuration <https://altair-viz.github.io/altair-tutorial/notebooks/08-Configuration.html>`_.
+        For example, you can (assuming you've done ``import altair as alt``):
+
+        - Change the width/height/title with
+          ``.properties(width=500, height=350, title="My amazing plot")``.
+        - Change the x-axis label rotation with ``.configure_axisX(labelAngle=30)``.
+        - Change the opacity of the points in your scatter plot with
+          ``.configure_point(opacity=.5)``.
+        - Set the x-axis title by passing ``x=alt.X('length', title='Length')`` instead
+          of ``x='length'``.
```

Comment:
This last bullet point feels incomplete:
Set the x-axis title by passing `x=alt.X(...)` instead of `x=...`... where? Where do I pass it?

(Also, you have the same list in `visualization.md` and in `series.py` but this fourth item is only present in this list. It looks like all three lists should look the same but I'm not sure, so I'm just flagging this here in case you missed it.)